src/languages/swift.js

```diff
@@ -7,39 +7,52 @@ Website: https://swift.org
-      // capture the keywords that start with the character "#"
-      $pattern: /[\w#]+/,
-      keyword: '#available #colorLiteral #column #else #elseif #endif #file ' +
-        '#fileLiteral #function #if #imageLiteral #line #selector #sourceLocation ' +
-        '_ __COLUMN__ __FILE__ __FUNCTION__ __LINE__ Any as as! as? associatedtype ' +
-        'associativity break case catch class continue convenience default defer deinit didSet do ' +
-        'dynamic dynamicType else enum extension fallthrough false fileprivate final for func ' +
-        'get guard if import in indirect infix init inout internal is lazy left let ' +
-        'mutating nil none nonmutating open operator optional override postfix precedence ' +
-        'prefix private protocol Protocol public repeat required rethrows return ' +
-        'right self Self set some static struct subscript super switch throw throws true ' +
-        'try try! try? Type typealias unowned var weak where while willSet',
-      literal: 'true false nil',
-      built_in: 'abs advance alignof alignofValue anyGenerator assert assertionFailure ' +
-        'bridgeFromObjectiveC bridgeFromObjectiveCUnconditional bridgeToObjectiveC ' +
-        'bridgeToObjectiveCUnconditional c compactMap contains count countElements countLeadingZeros ' +
-        'debugPrint debugPrintln distance dropFirst dropLast dump encodeBitsAsWords ' +
-        'enumerate equal fatalError filter find getBridgedObjectiveCType getVaList ' +
-        'indices insertionSort isBridgedToObjectiveC isBridgedVerbatimToObjectiveC ' +
-        'isUniquelyReferenced isUniquelyReferencedNonObjC join lazy lexicographicalCompare ' +
-        'map max maxElement min minElement numericCast overlaps partition posix ' +
-        'precondition preconditionFailure print println quickSort readLine reduce reflect ' +
-        'reinterpretCast reverse roundUpToAlignment sizeof sizeofValue sort split ' +
-        'startsWith stride strideof strideofValue swap toString transcode ' +
-        'underestimateCount unsafeAddressOf unsafeBitCast unsafeDowncast unsafeUnwrap ' +
-        'unsafeReflect withExtendedLifetime withObjectAtPlusZero withUnsafePointer ' +
-        'withUnsafePointerToObject withUnsafeMutablePointer withUnsafeMutablePointers ' +
-        'withUnsafePointer withUnsafePointers withVaList zip'
-    };
+  // https://docs.swift.org/swift-book/ReferenceManual/LexicalStructure.html#ID413
+  // https://docs.swift.org/swift-book/ReferenceManual/zzSummaryOfTheGrammar.html
+  const SWIFT_KEYWORDS = {
+    // Override the pattern since the default of /\w+/ is not sufficient to capture
+    // the keywords that start with a number sign (#) or that have parentheses.
+    $pattern: /#?\w+(\(\w+\))?/,
+    keyword:
+      // Reserved keywords. These require backticks to use as an identifier:
+      '_ Any as associatedtype break case catch class continue default defer deinit do else enum extension ' +
```

Comment:
Since we're reworking all this... please switch to using an array with one keyword per line and then `KEYWORD.join(" ")` that array as necessary... see `ecmascript.js` for examples, etc...  this is for ease of maintenance in the future.

Reply:
Will do!